src/Tools/dotnet-counters/CounterMonitor.cs

```diff
@@ -174,16 +200,11 @@ private async Task<int> StartMonitor()
             Task monitorTask = new Task(() => {
                 try
                 {
-                    var configuration = new SessionConfigurationV2(
-                        circularBufferSizeMB: 1000,
-                        format: EventPipeSerializationFormat.NetTrace,
-                        requestRundown: false,
-                        providers: Trace.Extensions.ToProviders(providerString));
-
-                    var binaryReader = EventPipeClient.CollectTracing2(_processId, configuration, out _sessionId);
-                    EventPipeEventSource source = new EventPipeEventSource(binaryReader);
-                    source.Dynamic.All += Dynamic_All;
-                    source.Process();
+                    RequestTracingV2(providerString);
+                }
+                catch (EventPipeServerException)
```

Comment:
We have a specific error for commands that aren't supported, I think we should check for it specifically